### PR TITLE
Implement and use standard post aggregate and proofs API

### PR DESCRIPTION
**Pull request by ajsutton**
## PR Description
Implement and use the standard post aggregate and proofs API (`/eth/v1/validator/aggregate_and_proofs`).

## Fixed Issue(s)
#2756 

## Documentation

- [x] I thought about documentation and added the `documentation` label to this PR if updates are required.

### CODE REVIEW

File: data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/PostAggregateAndProofs.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      final ValidatorDataProvider provider, final JsonProvider jsonProvider) {
+    this.jsonProvider = jsonProvider;
+    this.provider = provider;
+  }
+
+  @OpenApi(
+      path = ROUTE,
+      method = HttpMethod.POST,
+      summary = "Publish aggregate and proofs",
+      tags = {TAG_VALIDATOR},
```
Comment (rolfyone) [inline data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/PostAggregateAndProofs.java]: `TAG_V1_VALIDATOR, TAG_VALIDATOR_REQUIRED`

File: data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/PostAggregateAndProofs.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      requestBody =
+          @OpenApiRequestBody(content = {@OpenApiContent(from = SignedAggregateAndProof.class)}),
```
Comment (rolfyone) [inline data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/PostAggregateAndProofs.java]: if we can take an array in like below, we can set isArray = true (from mem) to tell the openapi it's an array

**Review by rolfyone — APPROVED** [review]
a couple of nits, maybe check swagger to make sure it renders ok once you change isArray = true...
mostly LGTM.

File: data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/PostAggregateAndProofs.java (hunk 2 above)
Comment (ajsutton) [reply data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/PostAggregateAndProofs.java]: Done.

File: data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/PostAggregateAndProofs.java (hunk 1 above)
Comment (ajsutton) [reply data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/PostAggregateAndProofs.java]: Done. Every time...

**Pull request merged**